code/modules/cm_marines/vehicle_part_fabricator.dm

```diff
@@ -58,6 +61,8 @@
 		return
 	visible_message(SPAN_NOTICE("[src] starts printing something."))
 	spend_point_store(cost)
+	if(part_type==/obj/structure/ship_ammo/sentry)
+		omnisentry_price +=omnisentry_price_scale
```

Comment:
the fact that we have to check the path (use `ispath()` instead of this) means that something is being done fundamentally wrong. Please change this to be generally in line with OOP.